tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
Mehh seems Ubuntu 1404 doesnt have the package either... This is one distro I think _might_ need it to work out of the box.